AlayaCare Cloud Tech Radar.csv

```diff
@@ -1,15 +1,16 @@
 "name","ring","quadrant","isNew","description"
+"Cypress","Adopt","languages-and-frameworks","FALSE","<p></p>"
+"Micro frontend","Trial","Techniques","FALSE","<p></p>"
```

Comment:
Weren't we following alphabetical order before ?

Reply:
This is to fix the ring order. their tool will define the ring in order of appearance in the file, so `Adopt` needs to appear first in the data.

Reply:
Shouldn't we regroup all categories together then ?